File: src/test/java/br/com/encrypt/CriptografiaTest.java
package br.com.encrypt;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.usuario.NovoCliente;

public class CriptografiaTest {

    @Test
    void argon2() {
        Criptografia.verificar(NovoCliente.novoCliente(), "teste123456");
    }
}

File: src/test/java/br/com/encrypt/CriptografiaTest.kt
package br.com.encrypt

import br.com.encrypt.Criptografia.verificar
import br.com.newObjects.model.usuario.NovoCliente
import org.junit.jupiter.api.Test

class CriptografiaTest {
    @Test
    fun argon2() {
        verificar(NovoCliente.novoCliente(), "teste123456")
    }
}
